sonic_db_config/db_config_test.go

```diff
@@ -79,23 +80,3 @@ func TestGetDbMultiNs(t *testing.T) {
 	})
 }
 
-func TestOverrideDbConfigFile(t *testing.T) {
```

Comment:
>TestOverrideDbConfigFile [](http://example.com/codeflow?start=5&length=24)

This UT is useful. Let's move it to swss-common repo?

Reply:
https://github.com/sonic-net/sonic-swss-common/blob/master/tests/main.cpp#L24-L34
swss-common has the unit test for nonexisting db config file.